test/test_torch.py

```diff
@@ -2661,9 +2661,14 @@ def test_qtensor(self):
         qr = r.quantize_linear(scale, zero_point)
         self.assertEqual(qr.q_scale(), scale)
         self.assertEqual(qr.q_zero_point(), zero_point)
+        self.assertTrue(qr.is_quantized)
         rqr = qr.dequantize()
         for i in range(num_elements):
             self.assertEqual(r[i], rqr[i])
+        # Testing item
```

Comment:
Can we check for actual quantization here (like comment for cpp test). We cannot check that r and rqr are identical as rqr will be the reduced precision equivalent of r.

Reply:
Yeah I did that in next diff